docs/MOTD.md

```diff
@@ -0,0 +1,362 @@
+# MOTD Protocol Documentation
+
+## Overview
+
+The server operates by listening on port `19132` for incoming RakNet unconnected pings. Upon receiving a valid ping, the server responds with an unconnected pong that contains the MOTD and other info. This interaction ensures that clients receive essential details about the server before establishing a connection.
+
+## Packet Structure
+
+### Unconnected Ping
+
+When a client wants to display the server it sends an unconnected ping
+
+- **Packet ID (1 byte):** `0x01`
```

Comment:
I wonder in general if we should use a schema like rust uses (`u32`) or more over the older C style (`unsigned int`) or something in between (`uint32`). I personally prefer something like `u32`...